src/app/containers/FrontPageMain/index.jsx

```diff
@@ -86,13 +79,8 @@ const FrontPageMain = ({ frontPageData }) => {
         <VisuallyHiddenText id="content" tabIndex="-1" as="h1">
           {offScreenText}
         </VisuallyHiddenText>
-        <FrontPageGrid columns={mainColumns} enableGelGutters>
-          <Grid
-            item
-            columns={itemColumns}
-            startOffset={startOffsets}
-            margins={itemMargins}
-          >
+        <FrontPageGrid columns={mainColumns} dir={dir} enableGelGutters>
+          <Grid item columns={itemColumns} dir={dir} margins={itemMargins}>
```

Comment:
We should consider removing this Grid item since it's the only grid item inside `FrontPageGrid` and it occupies 100% of the width at all breakpoints

Reply:
Good point but if we remove this Grid item the page will break as we won't be indicating how many columns should span the content within the `FrontPageGrid` which is an outher Grid.

<img width="1545" alt="Screenshot 2020-02-17 at 09 56 14" src="https://user-images.githubusercontent.com/46446236/74642977-e3989680-516b-11ea-927a-6349a4b9c92e.png">

Perhaps we should consider to not use neither Grid nor Grid item in the `FrontPageMain`, as they both have the same columns.

Reply:
Yes, changing `FrontPageGrid` to a styled div with margins applied to should work.

Reply:
This has been done in https://github.com/bbc/simorgh/pull/5139/files/854dcd319b9d1e0227d2241182ec9d3c90c60b90..3383bf4c9d7c20b84f00f4e6f9bbbd8f810eaddf

Reply:
@tochwill Yep, it turns out we don't need it at this level as the number of columns would be the same for the `FrontPageGrid` and its direct `Grid item` child. 
An alternative would have been to define them using just one column which is, in fact, the space they take at that level, but better to keep it as a regular `div`s with just `width` and `margins` applied.

Reply:
👍 This got updated during my review!